src/openzaak/management/commands/generate_data.py

```diff
@@ -75,8 +79,13 @@ def get_sl_resultaten() -> list[dict]:
     get first 100 objects from Selectielijst.resultaten endpoint
     use only for test purposes
     """
-    client = ReferentieLijstConfig.get_client()
-    response = client.list("resultaat")
+    config = ReferentieLijstConfig.get_solo()
+    client = build_client(config.service)
+
+    try:
+        response = client.get("resultaten").json()
+    except JSONDecodeError:
+        raise SelfTestFailed("Object type version didn't have any data")
```

Comment:
Why do you raise django_setup_configuration error here?

Reply:
no idea why this was here, I'll remove it